src/components/SingleFileInput/SingleFileInput.tsx

```diff
@@ -57,6 +58,9 @@ export type SingleFileInputProps = {
   /** The label to be displayed in the dropzone. */
   dropzoneLabel: string;
 
+  /** The button to be displayed in the dropzone. */
```

Comment:
@SarahDonvito Try to explain better this prop: is not the button per se (which is basically always included) but the label for the button